lib/xclarity_client/xclarity_validate.rb

```diff
@@ -0,0 +1,51 @@
+require 'json'
+
+module XClarityClient
+  class XClarityValidate < XClarityBase
```

Comment:
Validate doesn't look like a good name, once you are manipulating sessions. Validate is an Action, shouldn't be used as an Entity name.

Reply:
Firstly, Thank you for your review. I understand, but I create this class for be  used when the manageIQ check the credentials in the UI page. So, should I use XClarityValidation as name?

Reply:
I agree with you @walteraa, @CharlleDaniel Validation continuous being an action, inside this file you have many functions. XClaritySessions make more sense?

Reply:
If you use this class strictly to check credentials (not getting tokens, not creating cookie sessions, etc) you could use `XClarityCredentialsValidator`, it is important to make explicit what you are validating. Unless you saw the possibility of using this class to validate other things, so you could use `XClarityValidator`.

But I do not like the second Idea because using the second class-name suggestion to other validation contexts, you'll not respect the specialization design pattern.

Reply:
@walteraa done. Thank you for your suggestion. :cake: